packages/cozy-intent/src/api/models/methods.ts

```diff
@@ -12,6 +12,9 @@ export type NativeMethodsRegister = {
   openSettingBiometry: () => Promise<boolean>
   setFlagshipUI: (flagshipUI: FlagshipUI, caller?: string) => Promise<null>
   showSplashScreen: () => Promise<null>
+  toggleSetting: (
+    settingName: 'biometryLock' | 'PINLock' | 'autoLock'
+  ) => Promise<boolean | null>
```

Comment:
Why do we need `boolean | null` as return type? Shouldn't it be always `boolean`?

Reply:
It should return a boolean to confirm everything went well (even if we don't need the value since we have `flagshipMetadata` as a source of truth, and not an awaited value). But in case of error, null is the default return in async post-me.
You could make an argument that the boolean is useless and we should always use null return, but maybe we want to add an error handler web side, sometime down the road